samples/guide/src/main/java/okhttp3/recipes/kt/DevServer.kt

```diff
@@ -114,7 +51,7 @@ class DevServerAndroid {
   val hosts = arrayOf(server.hostName)
 
   val platformTrustManager = platformTrustManager()
-  val trustManager = AndroidAllowlistedTrustManager(platformTrustManager, *hosts)
+  val trustManager = OkHttpTrustManager.hostOverride(platformTrustManager, "localhost", InsecureTrustManager)
```

Comment:
I like how this sample ended up because we never implement the insecure mode ourselves, but give a clean "as safe as possible" sample given a concrete real need.